src/server/config.ts

```diff
@@ -19,3 +20,24 @@ export const ADYEN_ORIGIN_KEY = process.env.ADYEN_ORIGIN_KEY!
 export const ADYEN_ENVIRONMENT = process.env.ADYEN_ENVIRONMENT!
 
 export const APP_ENVIRONMENT = process.env.APP_ENVIRONMENT ?? 'development'
+
+const DEFAULT_FEATURES: FeatureMap = {
+  [Feature.CHECKOUT_CREDIT_CHECK]: ['NO'],
+  [Feature.CHECKOUT_UPSELL_CARD]: ['SE'],
+  [Feature.CUSTOMER_SERVICE_PHONE_NUMBER]: ['SE'],
+  [Feature.OFFER_PAGE_INSURANCE_TOGGLE]: ['SE'],
+  [Feature.QUOTE_CART_API]: [],
+  [Feature.TEST_FEATURE]: [],
+}
```

Comment:
@gustaveen here's some default values we can use if you haven't configured your local `.env`

Reply:
I feel like this defeats the point of "not having to touch the code to change feature flags. This section may age badly if we all at some point have local env files.

Reply:
Hmm... any env variable will override these defaults so in my mind we don't need to change code.

But perhaps it's like you say that it's mostly now in the beginning. I'm just thinking if we miss one feature flag and that feature accidentally gets turned off in production let's say. This would make sure that we have a default for stable features.

Reply:
That works IMO! 👌